editor/export_template_manager.cpp

```diff
@@ -401,7 +403,15 @@ void ExportTemplateManager::_http_download_templates_completed(int p_status, int
 				String path = download_templates->get_download_file();
 				template_list_state->set_text(TTR("Download Complete."));
 				template_downloader->hide();
-				_install_from_file(path, false);
+				int ret = _install_from_file(path, false);
+				if (ret) {
```

Comment:
Why not simply 
```C++
if (_install_from_file(path, false))
```
?

Reply:
Mostly personal preference for readability, but yes, what you propose could be used too.